semver/format_range_test.ts

```diff
@@ -61,7 +61,7 @@ Deno.test({
 
     for (const [r, expected] of versions) {
       await t.step({
-        name: `${r} -> ${expected}`,
+        name: `${r}`,
```

Comment:
```suggestion
        name: r,
```
Ditto for other instances in this PR where the string literal (?) just contain a string.

Reply:
Done